modules/EvseV2G/iso_server.cpp

```diff
@@ -1322,6 +1322,21 @@ static enum v2g_event handle_iso_charge_parameter_discovery(struct v2g_connectio
             req->DC_EVChargeParameter.EVMaximumVoltageLimit.Value < 0) {
             res->ResponseCode = iso2_responseCodeType_FAILED_WrongChargeParameter; // [V2G2-477]
         }
+
+        constexpr auto physical_value_to_float = [](const iso2_PhysicalValueType& pv) {
```

Comment:
Good catch. Have look into the EvseV2G if there is any similiar function

Reply:
I looked but I coudln't find a similar function